driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/IntelliJVersion.scala

```diff
@@ -3,5 +3,5 @@ package org.virtuslab.ideprobe.dependencies
 final case class IntelliJVersion(build: String) extends AnyVal
 
 object IntelliJVersion {
-  val Latest = IntelliJVersion("202.5792.28-EAP-SNAPSHOT")
+  val Latest = IntelliJVersion("202.6250.13-EAP-SNAPSHOT")
```

Comment:
what about the version of the plugin itself in `build.sbt`? Should it stay unchanged?

Reply:
I used buildinfo to have it in one place